.github/workflows/run-tests.yml

```
@@ -10,8 +10,9 @@ jobs:
 
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
       matrix:
-        python-version: [ 3.7, 3.8, 3.9 ]
+        python-version: [ 3.8, 3.9, 3.10 ]
```

```suggestion
        python-version: [ '3.8', '3.9', '3.10' ]
```

Now Python 3.1 is being tested